script/start.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -aueo pipefail
+
+source .env
+
+GOBIN=$(pwd)/bin
+
+echo -e "Cleanup: delete $GOBIN"
+rm -rf "$GOBIN"
+
+echo -e "Compiling..."
+go install -v .
+RESULT=$?
+if [ "$RESULT" -eq "0" ]; then
+    chmod -R 777 bin
+    echo -e "Build SUCCEEDED"
+else
+    echo -e "Build FAILED"
+    exit 1
+fi
+
+# Run
+./bin/azure-k8s-metrics-adapter \
+    --authentication-kubeconfig="$KUBE_CONFIG_FILE" \
+    --authorization-kubeconfig="$KUBE_CONFIG_FILE" \
+    --lister-kubeconfig="$KUBE_CONFIG_FILE" \
+    --secure-port=6443 \
+    --logtostderr=true \
+    --v 5 #\
+    #--secure-port 80
```

Comment:
Same comment above around improving the contribution docs.  This script is helpful as it also shows how to run outside of the cluster with the kubeconfig but would like to push folks to using the makefile as it has some of test and build built in